tools/_venv_common.py

```diff
@@ -158,6 +158,8 @@ def main(venv_name, venv_args, args):
                           env=new_environ, shell=True)
     subprocess_with_print('python {0} {1}'.format('./tools/pip_install.py', ' '.join(args)),
                           env=new_environ, shell=True)
+    subprocess_with_print("python -m pip install --upgrade 'setuptools>=30.3'",
+                          env=new_environ, shell=True)
```

Comment:
I think it may be a good idea to do this before calling `pip_install.py` as it should avoid the problem we hit with `pytest` when installing packages there as well.